pull_requests.md

```diff
@@ -51,6 +80,31 @@ best in __Refactoring: Improving the Design of Existing Code__:
 
 > “Any fool can write code that a computer can understand. Good programmers write code that humans can understand.“
 
+* Make in you comments concrete improvement suggestions with a clear rationale for the suggested change.
+  Keep comments polite and free of possibly insulting statements.
+  Remember that written language lacks emotional context and body language, and is easily misunderstood.
+* Keep discussions of suggested changes short, avoid more than 2-3 cycles of request-response; if the suggested change is controversial consider a direct discussion of the topic
+* the tech lead has the last word in disagreements
+* things to look out for in a code review
+  * Is every piece of code in the right place, i.e. model code in a model, controller logic in a controller, app-specific helper code in a helper, generic helper code in a library?
+  * Do all classes have only one responsibility?
+  * Do all methods do only one thing?
+  * If a method has a side-effect, is that clear from the name, and otherwise documented?
+  * Are all classes/methods/variables named properly so that the code is self-describing?
+  * Is everything as private as possible, i.e. only the intended public interface is public?
+  * Are too many things private? This could be an indication that you should extract a class.
+  * Are all files within a reasonable size (e.g., less than 100 lines of code)?
```

Comment:
In my experience, clear guidelines what to do and not to do is what junior devs want, to at least get started with something. They can fine-tune their understanding along the way.